go.mod

```diff
@@ -21,8 +21,8 @@ require (
 	github.com/IBM/ibm-hpcs-tke-sdk v0.0.0-20211109141421-a4b61b05f7d1
 	github.com/IBM/ibm-hpcs-uko-sdk v0.0.4
 	github.com/IBM/keyprotect-go-client v0.7.0
-	github.com/IBM/networking-go-sdk v0.34.0
-	github.com/IBM/platform-services-go-sdk v0.28.5
+	github.com/IBM/networking-go-sdk v0.33.0
```

Comment:
Can you rebase with the latest go.mod and go.sum since the master is need github.com/IBM/networking-go-sdk v0.34.0
not 	github.com/IBM/networking-go-sdk v0.33.0

Reply:
Gotcha, updated the go.mod file and it looks like it's passing now